components/identity/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -71,6 +71,8 @@
     public final static String OAUTH_USER_NAME = "oauth_user_name";
     public final static String OAUTH_ACCESS_TOKEN_ISSUED = "oauth_access_token_issued";
 
+    public final static String TOKEN_TYPE = "token_type";
```

Comment:
There should be a constant available for this. Check constants in apache amber package.